Support clearing devices keys of exported TTNv2 devices (#13)
TheThingsNetwork/lorawan-stack-migrate: README.md

$ export TTNV2_DISCOVERY_SERVER_ADDRESS="discovery.thethings.network:1900"
```

### Notes

> - The export process will halt if any errors occur. Execute any commands with the `--dry-run` flag first, to verify their result.

Reviewer: ```suggestion
- The export process will halt if any error occurs.
- Execute commands with the `--dry-run` flag to verify if the outcome will be as expected.
```

$ export TTNV2_DISCOVERY_SERVER_ADDRESS="discovery.thethings.network:1900"
```

### Notes

- The export process will halt if any errors occur. Execute any commands with the `--dry-run` flag first, to verify their result.
- Payload formatters are not exported. See [Payload Formatters](https://thethingsstack.io/integrations/payload-formatters/).
- Active device sessions are exported by default. You can disable this by using the `--ttnv2.with-session=false` flag. It is recommended that you do not export session keys for devices that can instead re-join on The Things Stack.

> - **IMPORTANT**: The migration from The Things Network Stack V2 to The Things Stack is one-way. Note that it is crucial that devices are handled by one Network Server at a time. In the commands below, we use the `--ttnv2.clear-device-keys=true` flag, which will clear both the root keys (**AppKey**, if any) the session keys (**AppSKey**, **NwkSKey**) from The Things Network Stack V2 after exporting them. Make sure you understand the ramifications of this. It is possible to keep the session keys on The Things Network Stack V2, by passing `--ttnv2.clear-device-keys=false`. **Note that having the session keys present on both Network Servers is not supported, and you will most likely encounter uplink/downlink traffic issues and/or a corrupted device MAC state**.

Reviewer: Okay so `clear-device-keys` is by default `false`, but `dry-run` is by default also `false`?

Why don't we have one flag, `dry-run`, and if it's set, don't clear anything? And let `with-session` indicate what to clear and export.

So basically

`--dry-run` | `--with-session` | What happens
--- | --- | ---
`false` | `false` | reset root key, export root key but not session
`false` | `true` | reset session keys and root key, export all
`true` | `false` | no reset, export root key but not session
`true` | `true` | no reset, export all

And then export with:

```bash
# dry run first, verify that no errors occur

> $ ttn-lw-migrate devices --source ttnv2 --ttnv2.clear-device-keys=true "mydevice" --dry-run --verbose < device_ids.txt > devices.json

Reviewer: This is where I find `clear-device-keys` confusing with `dry-run`.
